swagger/voice_calls/voice_calls.json

```diff
@@ -0,0 +1,733 @@
+        "security": [{ "Bearer": [] }]
+      }
+    },
+    "/v3/communication/voice_calls/{uid}": {
+      "get": {
+        "operationId": "VoiceCallsController_get",
+        "parameters": [{ "name": "uid", "required": true, "in": "path", "schema": { "type": "string" } }],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response"},
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/VoiceCallResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["voice-calls"],
+        "security": [{ "Bearer": [] }]
+      },
+      "put": {
+        "operationId": "VoiceCallsController_update",
+        "parameters": [{ "name": "uid", "required": true, "in": "path", "schema": { "type": "string" } }],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": { "schema": { "$ref": "#/components/schemas/UpdateVoiceCall" } }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response" },
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/VoiceCallResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["voice-calls"],
+        "security": [{ "Bearer": [] }]
+      }
+    },
+    "/v3/communication/voice_calls/recordings/{uid}": {
+      "delete": {
+        "operationId": "VoiceCallsController_deleteVoiceCallRecording",
+        "parameters": [{ "name": "uid", "required": true, "in": "path", "schema": { "type": "string" } }],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response" },
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/VoiceCallResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["voice-calls"],
+        "security": [{ "Bearer": [] }]
+      }
+    },
+    "/v3/communication/quotas": {
+      "post": {
```

Comment:
This is a voice_call_quota. Please adjust accordingly. the end point should be /v3/communication/voice_call_quotas

Reply:
@nikitarablus @goldi-tz